test/sqllogictest/cluster_log_drop.slt

```diff
@@ -23,7 +23,7 @@ reset-server
 query T rowsort
 SELECT (SELECT COUNT(*) FROM mz_catalog.mz_sources WHERE name NOT LIKE '%_1' AND name NOT LIKE '%_2' AND name NOT LIKE '%_3') - (SELECT COUNT(*) FROM mz_catalog.mz_sources WHERE name LIKE '%_1');
 ----
-4
+5
```

Comment:
yeah, I'm not loving that we have to keep bumping this ... 🙈 

at some point i'll be fed up enough to look into fixing this in a better way... 😅

Reply:
yeah me too lol